image_captioning/models/encoder/encoder_test.py

```diff
@@ -0,0 +1,17 @@
+"""Simple test code for the Encoder module
+"""
+import tensorflow as tf
+
+from .encoder import Encoder
+
+# Test Encoder
+encoder = Encoder(model_name='inception_v3', embedding_dim=64)
+print(f'Encoder:\n{encoder}\n')
+
+input_image = tf.random.uniform(shape=(1, 299, 299, 3), dtype=tf.float32)
+print(f'Input Image Shape: {input_image.shape}')
+print(f'Input Image: {input_image}\n')
+
+output = encoder(input_image)
+print(f'Output Feature Shape: {output.shape}')
+print(f'Output Feature: {output}\n')
```

Comment:
Probably assert if the output shape is the same as the expected output shape?

Reply:
here's a reference: https://github.com/dansuh17/image-captioning/pull/14/files#diff-aac9c1874ae72cd908772ce8feb8795fc36bd2e8c66d2432147a11acf311e4fe

Reply:
Will refer to that 👍